src/apps/api/serializers/competitions.py

```diff
@@ -121,6 +121,7 @@ class CompetitionDetailSerializer(serializers.ModelSerializer):
     phases = PhaseDetailSerializer(many=True)
     leaderboards = LeaderboardSerializer(many=True)
     collaborators = CollaboratorSerializer(many=True)
+    participant_status = serializers.CharField(read_only=True)
```

Comment:
should have a `choices=` option here for validation up-and-down the Django stack?